### PR TITLE
Task-50578: Adapt documents mobile view to design.

### PR DESCRIPTION
**Pull request by mdiha**

### CODE REVIEW

File: documents-webapp/src/main/webapp/skin/less/documents.less [hunk 1]
```diff
@@ -62,18 +67,33 @@
           font-size: 32px !important;
         }
 
-        span {
-          display: flex;
-          margin-left: 16px !important;
+        .document-title {
+          display: inline-flex;
```
Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less]: You can Add directly the vuetify class  **d-inline-flex**

File: documents-webapp/src/main/webapp/skin/less/documents.less [hunk 2]
```diff
@@ -62,18 +67,33 @@
...
+          width: 28vh;
+
+          .document-name {
+            margin-left: 16px !important;
```
Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less]: You can Add directly the vuetify class **ms-4**

File: documents-webapp/src/main/webapp/skin/less/documents.less [hunk 3]
```diff
@@ -62,18 +67,33 @@
...
+          }
+
+          .document-type {
+            margin-left: 0 !important;
```
Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less]: You can Add directly the vuetify class **ms-0**

File: documents-webapp/src/main/webapp/skin/less/documents.less [hunk 4]
```diff
@@ -1,7 +1,11 @@
 @import "./variables.less";
 
 .VuetifyApp {
-  .documents-application {
+  #DocumentsApplication {
+    .document-time {
+      color: @textLightColor;
```
Comment (SaraBoutej) [inline documents-webapp/src/main/webapp/skin/less/documents.less]: You can Add directly the class **text-light-color**

**Review by SaraBoutej — CHANGES_REQUESTED** [review]

File: documents-webapp/src/main/webapp/skin/less/documents.less (hunk 1 above)
Comment (mdiha) [reply documents-webapp/src/main/webapp/skin/less/documents.less]: Done

**Review by SaraBoutej — APPROVED** [review]
LGTM

**Pull request merged**